angular/src/app/landing/sections/resourcedata.component.ts

```diff
@@ -74,8 +88,10 @@ export class ResourceDataComponent implements OnChanges {
      *                  full list from the NERDm Resource record 
      */
     selectAccessPages(comps : NerdmComp[]) : NerdmComp[] {
-        let use : NerdmComp[] = comps.filter(cmp => cmp['@type'].includes("nrdp:AccessPage") &&
-                                       ! cmp['@type'].includes("nrd:Hidden"));
+        let use : NerdmComp[];
+        use = comps.filter(cmp => cmp['@type'].includes("nrdp:AccessPage") &&
+        ! cmp['@type'].includes("nrd:Hidden"));
```

Comment:
Ok, @chuanlin2018, I think this is what we should do to avoid the redundant display.  It's based on the idea that a "theme" is a named collection of display preferences.  After line 93, we can say, 
```
    if (this.theme == Themes.SCIENCE_THEME) 
        use = use.filter(cmp => ! cmp['@type'].includes("nrda:DynamicResourceSet"));
```
(I'm not sure how close this is what you had before.)

Reply:
Okay. No change to html part is necessary then, correct?